astroquery/nrao/core.py

```diff
@@ -144,8 +143,12 @@ def _gen_sql(payload):
     return sql + where
 
 
-class NraoAuth(BaseVOQuery, BaseQuery):
-    pass
+# class NraoAuth(BaseVOQuery, BaseQuery):
+#     """
+#     TODO: this needs to be implemented
+#     """
+#     pass
+
 
 class NraoClass(BaseQuery):
```

Comment:
I believe you still want to inherit this from `BaseVOQuery`, so the session things passed through pyvo correctly.